lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -258,6 +259,29 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
   defp format_short_route_pill(route_id),
     do: route_id |> String.first() |> String.downcase() |> Kernel.<>("l")
 
+  # Alert subheaders should not wrap in the middle of a station name
+  # so we have to use FreeTextLines to prevent the wrapping.
+  # This function takes a list of proper noun strings and
+  # returns a list of FreeTextLines with "nowrap" applied
+  @spec format_station_name_list([String.t()]) :: list(FreeText.t())
+  defp format_station_name_list([string]), do: [%{format: :nowrap, text: "#{string}"}]
+
+  defp format_station_name_list([s1, s2]),
+    do: [
+      %{format: :nowrap, text: "#{s1}"},
+      " & ",
+      %{format: :nowrap, text: "#{s2}"}
+    ]
+
+  defp format_station_name_list(list) do
+    list
+    |> List.update_at(length(list) - 1, &" & #{&1}")
+    |> Enum.join(", #")
+    |> String.split("#")
+    |> Enum.map(fn string -> %{format: :nowrap, text: "#{string}"} end)
```

Comment:
Should be able to use `string` directly, since it's already a string:
```suggestion
    |> Enum.map(fn string -> %{format: :nowrap, text: string} end)
```

Reply:
PFFT ha good catch 😅